### PR TITLE
Menu navigation bar

### LINKED ISSUE #733: Add shared menu bar
The shared menu bar should allow users to easily switch between applications

### PR DESCRIPTION
**Pull request by aguspina**
Fixes #733

### CODE REVIEW

File: config/locales/es.yml [hunk 1]
```diff
@@ -94,6 +96,7 @@ es:
   not_found: ¡La página que buscaste no existe!
   not_found_explanation: '¡Ups! Esto es lo que se conoce como %{error}: fijate si escribiste bien la dirección. Peeeeero, si pensás que es un error nuestro, %{issues}'
   notify_problem_with_exercise: ¿Pensás que algo está mal con este ejercicio?
+  office: Oficina
```
Comment (flbulgarelli) [inline config/locales/es.yml]: Secretaría?

**Comment by flbulgarelli** [comment]
Could you provide an screenshot?

**Comment by flbulgarelli** [comment]
PS: I'd like the menu bar to have icons.

**Comment by aguspina** [comment]
@flbulgarelli  Please review again

**Comment by aguspina** [comment]
screen shot 2017-01-26 at 7 02 35 pm

**Comment by flbulgarelli** [comment]
Nice. But, we should only show links to applications when they have permissions to do that.

**Pull request merged**